File: ACL/backend/rest-api/model/folderpermission.go
package model

type FolderPermission struct {
	Id            int64  `json:"u_id,string,omitempty" key:"primary" column:"u_id"`
	FId           int64  `json:"folder_id,string,omitempty" column:"folder_id"`
	PType         string `json:"p_type" column:"p_type"`
}

func (folderpermission *FolderPermission) Table() string {
	return "folder_permission"
}


func (folderpermission *FolderPermission) String() string {
	return Stringify(folderpermission)
}

File: ACL/frontend/ACLProject/src/app/folderpermission.ts
export class FolderPermission {
    public u_id: number;
    public folder_id: number;
    public p_type: string;
    constructor(u_id: number,folder_id:number,p_type: string)
    {
    this.u_id=u_id;
    this.folder_id=folder_id;
    this.p_type=p_type;
    }
 }